website/project/model.py

```diff
@@ -665,17 +665,25 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
         {
             'unique': False,
             'key_or_list': [
+                ('date_modified', pymongo.DESCENDING),
+            ]
+        },
+        {
+            'unique': False,
+            'key_or_list': [
                 ('tags.$', pymongo.ASCENDING),
                 ('is_public', pymongo.ASCENDING),
                 ('is_deleted', pymongo.ASCENDING),
+                ('is_institution', pymongo.ASCENDING),
```

Comment:
As a heads up, `is_institution` will be removed with https://github.com/CenterForOpenScience/osf.io/pull/5228 , which will be merged soon.